src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -20,9 +43,158 @@ public class MauiMediaElement : Grid, IDisposable
 	public MauiMediaElement(MediaPlayerElement mediaPlayerElement)
 	{
 		this.mediaPlayerElement = mediaPlayerElement;
+
+		btn = new Button
+		{
+			Background = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
+			Foreground = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Transparent),
+			Width = 40,
+			Height = 40
+		};
+		btn.Click += ButtonSetFullScreenStatus;
+
+		Image image = new()
+		{
+			Width = 40,
+			Height = 40,
+			HorizontalAlignment = Microsoft.UI.Xaml.HorizontalAlignment.Right,
+			VerticalAlignment = Microsoft.UI.Xaml.VerticalAlignment.Top,
+			Source = new BitmapImage(new Uri("ms-appx:///whitefs.png")),
```

Comment:
If we cache this `BitmapImage` and reuse, it will work as expected?

Reply:
During testing there were no issues. I set full screen and restored previous state. Back and forth a few times with zero issues. I was using sample app.

Reply:
great, so let's cache the image instead of creating a new one everytime